benchmark/bench_CQRRPT/CQRRPT_speed_comparisons.cc

```diff
@@ -66,7 +66,7 @@ static std::vector<long> call_all_algs(
     auto d_factor = all_data.sampling_factor;
 
     // Additional params setup.
-    RandLAPACK::CQRRPT<double, r123::Philox4x32> CQRRPT(true, true, tol);
+    RandLAPACK::CQRRPT_GPU<double, r123::Philox4x32> CQRRPT(true, true, tol);
```

Comment:
I'm skeptical that this benchmark should assume GPU support.

Reply:
Yes, in fact, benchmarks were not touched in this project at all.